eiffel-vocabulary/EiffelIssueDefinedEvent.md

```diff
@@ -0,0 +1,278 @@
+<!---
+   Copyright 2018 Jaden Young
+   For a full list of individual contributors, please see the commit history.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+--->
+
+# EiffelIssueDefinedEvent (ID)
+The EiffelIssueDefinedEvent declares that an issue has been created in some
+external issue management software. This event provides provenance for an issue,
+and acts as a "handle" for future issue related events similarly to
+[EiffelActivityTriggeredEvent](./EiffelActivityTriggeredEvent.md) and
+[EiffelTestCaseTriggeredEvent](./EiffelTestCaseTriggeredEvent.md). It provides
```

Comment:
I don't think it is fair to compare ID with ActT and TCT. The latter events represent that an operation is put in queue to be executed. I would propose you end the sentence after 'issue related events'. It is semantically quite similar to ED and FCD though.

Reply:
The description could definitely use some clarification. ID is certainly
similar semantically to FCD/ED; an issue really is just a specific kind of
flow context with additional data attached. What motivated the comparison to
ActT/TCT was the inclusion of state intrinsic to the entity. If we think
about an issue being defined, work being done on an issue, and being
resolved, it's quite similar to ActT, ActS, and ActF. We could say that ID
puts an operation (resolve this) on a queue (backlog) to be executed (by a
human). However, the similarity to ActT/TCT is moot without events like ISM.